samples/test_relocation/prj.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_CODE_DATA_RELOCATION=y
+CONFIG_HAVE_CUSTOM_LINKER_SCRIPT=y
+CONFIG_CUSTOM_LINKER_SCRIPT="linker_arm_sram2.ld"
```

Comment:
We must have a sample that does not require copying the cortex linker script.

The use-case of this functionality is that one doesn't need to maintain a lot of linker script code, but if we need to maintain an entire copy of the linker script, then we are not achieving this.

Reply:
the only reason this is needed is because we are creating new memory regions to showcase this feature. if someone is using this feature they will have their own linker script in SOC anyway.

This unfortunately is a must to showcase an example.

Reply:
Couldn't we use some platform that already has multiple regions?

Then we wouldn't need to create new ones.

IMHO the cost of maintaining another copy of the cortex linkerscript is greater than the benefit we get from the test coverage of CONFIG_CODE_DATA_RELOCATION.

Reply:
>IMHO the cost of maintaining another copy of the cortex linkerscript is greater than the benefit we get from the test coverage of CONFIG_CODE_DATA_RELOCATION.

@SebastianBoe  Makes sense. I found a workaround. Please re-review.

Reply:
This is OK. Resolving this point.